.github/workflows/ci.yml

```diff
@@ -13,11 +13,17 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - uses: gradle/wrapper-validation-action@v1
+
       - uses: actions/setup-java@v4
         with:
           distribution: 'temurin'
           java-version: '17'
 
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '3.2'
+          bundler-cache: true
```

Comment:
Ruby needed for `:website:browserBuild` task (that is run below) which runs Jekyll for the markdown → website conversion.